wagtail/documents/templates/wagtaildocs/chooser/upload_form.html

```diff
@@ -1,5 +1,11 @@
 {% load i18n wagtailadmin_tags %}
-<section id="upload" class="{% if form.errors %}active {% endif %}nice-padding">
+<section
+    id="tab-upload"
+    class="w-tabs__panel w-transition w-duration-150 w-translate-y-4 w-opacity-0"
```

Comment:
Not sure what is going on here but for me this leads to the tabpanel being invisible. Are those styles needed? IF so could we set them based on the ARIA attributes, or failing this via the JS only?

Reply:
Removing these extra utils did the trick 👍